projects/ad74413r/builds.json

```diff
@@ -0,0 +1,15 @@
+{
+  "stm32": {
+    "dummy_example": {
+      "flags" : "DUMMY_EXAMPLE=y IIO_EXAMPLE=n IIO_TRIGGER_EXAMPLE=n"
```

Comment:
are IIO_EXAMPLE=n IIO_TRIGGER_EXAMPLE=n flags needed?

Reply:
They are not. Will add them back once the IIO example is implemented.